templates/web/base/common_header_tags.html

```diff
@@ -6,6 +6,10 @@
     id="js-meta-data" data-page="[% page %]"
     data-cobrand="[% c.cobrand.moniker %]">
 
+[% IF csrf_token %]
+  <meta content="[% csrf_token %]" name="csrf-token" />
```

Comment:
Sorry, just picking this up. The lines that I think you're talking about (https://github.com/mysociety/fixmystreet/pull/1652/files#diff-edf8ca8647028bd9be9b2f3f92e67ab6L73 and https://github.com/mysociety/fixmystreet/pull/1652/files#diff-0c932d62c9dad1e14d7c9e8bde78d4f5R376) are dealing with `jqXHR` objects, so to parse these, I'd have to load the `responseText` into a jQuery object, which seems like overkill. I think a regex is probably fine here (unless I've got the wrong end of the stick!)